src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php

```diff
@@ -10,17 +10,17 @@
 interface AkeneoResourcePageInterface
 {
     /**
-     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
+     * @return $this
      */
-    public function getFirstPage(): AkeneoResourcePageInterface;
+    public function getFirstPage();
 
     /**
-     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface|null
+     * @return $this|null
      */
     public function getPreviousPage(): ?AkeneoResourcePageInterface;
```

Comment:
I don't know if we need to specify return type here because of `$this`.

Reply:
because of null or $this might be returned